google-cloud-storage/src/test/java/com/google/cloud/storage/ITJsonResumableSessionPutTaskTest.java

```diff
@@ -110,51 +110,6 @@ public void emptyObjectHappyPath() throws Exception {
     }
   }
 
-  /**
-   *
-   *
-   * <h4>S.9</h4>
-   *
-   * Partial successful append to session
-   *
-   * <p>The client has sent N bytes, the server confirmed N bytes as committed. The client sends K
-   * bytes starting at offset N. The server responds with only N + L with 0 &lt;= L &lt; K bytes as
-   * committed.
-   */
-  @Test
-  public void scenario9() throws Exception {
```

Comment:
This test would verify that GCS ack'ing fewer bytes than were sent was correctly categorized as a failure scenario, however as we are now gracefully handling the partial write case. We no longer need this test for failure categorization.